elements/pfe-sass/extends/_typography_extends.scss

```diff
@@ -0,0 +1,55 @@
+/// EXTENDS
+/// @see {mixin} pfe-text 
+
+%pfe-content-base {
+    font-family: pfe-var(font-family);
+    line-height: pfe-var(line-height);
+    font-weight: pfe-var(font-weight--light);
+}
+
+/// CONTENT BLOCK
+%pfe-content--p {
+    @include pfe-text(md); 
+}
+
+%pfe-content--text--spacing-block {
+    &:not(:last-child):not(:empty) {
+        margin-bottom: pfe-var(content-spacer--body--md); //16px
+    }
+} 
+// These are not included in the content placeholders below because they come 
```

Comment:
If you use the triple / syntax `///` it will add it to the sass doc...unless you didn't want it in the sassdoc on purpose?